src/mmw/mmw/settings/layer_settings.py

```diff
@@ -89,6 +89,9 @@
         'url': 'https://s3.amazonaws.com/com.azavea.datahub.tms/'
                'nlcd/{z}/{x}/{y}.png',
         'overlay': True,
+        'maxNativeZoom': 13,
+        'maxZoom': 18,
+        'opacity': 0.618,
```

Comment:
It is 1 / $\phi$ and also 1 - $\phi$, the golden ratio.  It does not have any special significance beyond that.